pkg/controller/servicebindingrequest/nested/merge_test.go

```diff
@@ -0,0 +1,71 @@
+package nested
+
+import (
+	"testing"
+
+	"github.com/imdario/mergo"
+	"github.com/stretchr/testify/require"
+)
+
+func TestMerge(t *testing.T) {
```

Comment:
we should not write tests that verify the behaviour of imported libraries.

Reply:
Removed.